modular_skyrat/modules/syndie_silicon/code/interdyne_emag.dm

```diff
@@ -0,0 +1,8 @@
+/obj/item/card/emag/interdyne
+	name = "interdyne silicon card"
+	special_desc = "A modified ID card used to override silicon for use on the DS-2 and Interdyne stations. Do not bite."
+
+/obj/item/card/emag/interdyne/can_emag(atom/target, mob/user)
+	if(ispath(target.type, /mob/living/silicon/ai))
```

Comment:
convert this to an `is_type_in_list` after making the types you can emag a var on the emag